binary_parser.go

```diff
@@ -125,7 +123,7 @@ func (bp *binaryParser) parseObjectRef(index uint64) (val *plistValue, err error
 	case 0xd: // dictionary
 		return bp.parseDict(marker)
 	}
-	return nil, fmt.Errorf("unkown object type %x", marker>>4)
+	return nil, fmt.Errorf("unknown object type %x", marker>>4)
```

Comment:
add `plist: `in front of the error, so that a log would read
`plist: unknown object ...`